proto/eni.proto

```diff
@@ -22,17 +22,14 @@ message Eni {
     // Vnet that ENI belongs to
     string vnet = 6;
     // Optional
-    // Private Link encoding for IPv6 SIP transpositions
-    optional types.IpPrefix pl_sip_encoding = 7;
```

Comment:
Lets keep it here. Some bits seems to be required for datapath. We can update HLD later. @r12f for viz

Reply:
yea, in the latest design the pl_sip_encoding needs to be kept.

also changing the id for existing fields might not be a good idea, as it breaks ABI : D

Reply:
deprecated this field instead of deleted